app/crud.py

```diff
@@ -50,39 +44,38 @@ def create_art_item(db:Session, dataset_id:int, art_item: ItemSchema):
     _art_item = Item(id =str(uuid.uuid4()),
                     _data = art_item._data,
                     dataset = dataset,
-                    geometry = art_item.geometry) 
+                    geometry = art_item.geometry)
     db.add(_art_item)
     db.commit()
     db.refresh(_art_item)
     return _art_item
 
-# Remove art_item data 
+
 def remove_art_item(db:Session, art_item_id:int):
     """_art_item = get_art_item_by_id(db=db, art_item_id = art_item_id)
     db.delete(_art_item)
     db.commit()
     """
     pass
 
-# update art_item data
-def update_art_item(db:Session,
-                    dataset_id:int,
+
+def update_art_item(db: Session,
```

Comment:
here you no longer pass a dataset_id, is it a part of art_item schema or why you skip this parameter?

Reply:
yep you already have it as part of the schema, no?